src/View/SSViewer.php

```diff
@@ -165,7 +165,10 @@ public static function set_themes($themes = [])
 
     public static function add_themes($themes = [])
     {
-        SSViewer::config()->merge('themes', $themes);
+        $currentThemes = SSViewer::get_themes();
+        $finalThemes = array_merge($themes, $currentThemes);
+        SSViewer::config()->remove('themes');
```

Comment:
`->remove()` is redundant, since `->set_themes()` uses `->set()` inside anyway.

Totally valid if it was using `->merge()` inside.

Reply:
Thanks, have updated both this and the test 👍